app/serializers/organization_page_serializer.rb

```diff
@@ -0,0 +1,36 @@
+class OrganizationPageSerializer
+  include RestPack::Serializer
+  include CachedSerializer
+
+  attributes :id, :href, :created_at, :updated_at, :url_key, :title,
+    :language, :content, :type
+
+  can_include :organization
+  can_filter_by :url_key, :language
+
+  def self.page_href(page, options)
```

Comment:
this is pretty much a straight copy coudn't we extract the code to a module that reflected on the resource class / id and just include that to get the behaviour? understand if you'd prefer not to